plugins/Files/js/global.js

```
@@ -94,20 +94,17 @@ $('.dropdown .button').click(function() {
 			break;
 		default:
 			console.error('Unknown button!', this);
-			return; // Don't close dropdown
-	}
-
-	// Detect flawed userInput from actions that require it, hide if fine
-	if (!userInput && option !== 'Make Folder') {
-		tools.tooltip('Invalid action!', this);
```

If no input, silently fail and close the dropdown? Is that better?